uglyscript.js

```diff
@@ -30,7 +35,7 @@ function getImageFromElements(elements) {
             return mainImage.file
         }
     }
-    return null;
+    return "https://media.giphy.com/media/l4KibK3JwaVo0CjDO/giphy.gif";
```

Comment:
you could store this in a var or a const or something, maybe a sensible name like catToboganningDownTheStairs 🐈